packages/client/ui/react-ui/src/hooks/useCrossmint.tsx

```diff
@@ -33,17 +33,17 @@ export function CrossmintProvider({
         })
     );
 
-    const setJwt = (jwt: string | undefined) => {
+    const setJwt = useCallback((jwt: string | undefined) => {
```

Comment:
@MatiNavCrossmint had to add them back because when `useCrossmint` is re-rendered, `setJWT` would change and wherever `setJWT` is used as a dependency would be re-executed.